regler/src/main/kotlin/no/nav/pleiepengerbarn/uttak/regler/BeregnGrader.kt

```diff
@@ -96,6 +96,15 @@ internal object BeregnGrader {
         val restTilSøker =
             finnRestTilSøker(beregnGraderGrunnlag.pleiebehov, etablertTilsynprosent, beregnGraderGrunnlag.andreSøkeresTilsyn, beregnGraderGrunnlag.overseEtablertTilsynÅrsak)
 
+        if (beregnGraderGrunnlag.overstyrtInput != null) {
```

Comment:
Her bør vi tillate at uttaksgrad ikke blir overstyrt. Dette skjer for eksempel når saksbehandler kun vil overstyre utbetalingsgraden for ett gitt arbeidsforhold til 0%.

For eksempel, endre til:
```
    beregnGraderGrunnlag.overstyrtInput?.overstyrtUttaksgrad?.let {
        return UttaksgradResultat(
                restTilSøker,
                it,
                oppfyltÅrsak = Årsak.AVKORTET_MOT_INNTEKT,
                overseEtablertTilsynÅrsak = beregnGraderGrunnlag.overseEtablertTilsynÅrsak
            )
    }
```